Gemfile

```diff
@@ -16,6 +16,14 @@ install_if -> { RUBY_PLATFORM !~ /darwin/ } do
   gem "openssl", "= 3.2.0"
 end
 
+if RUBY_PLATFORM =~ /aix/ && RUBY_VERSION < "3.1"
+  gem "mixlib-log", ">= 2.0.3", "<= 3.1.1"
+else
+  gem "mixlib-log", ">= 3.2.0", "< 4.0"
+end
+
+gem "rdoc", "~> 6.4.1" # 6.4.1.1 required for CVE-2024-27281, allow patch upgrades
```

Comment:
Is the `rdoc` change supposed to be here? Was removed specifically https://github.com/chef/chef/pull/14857/files

Reply:
Weird, that reference is pulled. Nothing else related to rdoc was in there.